clarity/src/vm/types/signatures.rs

```diff
@@ -1578,7 +1827,7 @@ mod test {
             epoch,
         )
         .unwrap()[0];
-        TypeSignature::parse_type_repr(expr, &mut ()).unwrap_err()
+        TypeSignature::parse_type_repr(StacksEpochId::latest(), expr, &mut ()).unwrap_err()
```

Comment:
Also flagging this to use each epoch version for testing.

Reply:
fixed.